editor/import/resource_importer_scene.cpp

```diff
@@ -1256,7 +1255,7 @@ Error ResourceImporterScene::import(const String &p_source_file, const String &p
 		Object::cast_to<Spatial>(scene)->scale(Vector3(root_scale, root_scale, root_scale));
 	}
 
-	scene->set_name(p_options["nodes/root_name"]);
```

Comment:
I don't think that we should drop the option (beyond the fact that it breaks compat).

It would be better to do something like:
```
p_options["nodes/root_name"] != "Scene Root" ? p_options["nodes/root_name"] : p_save_path.get_file().get_basename()
```
(or maybe formatted as an `if`/`else` as the ternary operator might not be very readable)

Reply:
I recently noticed the root name option in import settings and using it actually makes sense. Changed.